tailwind.config.js

```diff
@@ -31,6 +31,10 @@ module.exports = {
           900: "#192040",
         },
       },
+      width: {
+        323: "80.9rem",
+        318: "79.6rem",
+      },
     },
```

Comment:
No need to update the tailwind config. The tailwind config should be kept intact.

Reply:
Ok I have reverted those changes.